src/main/java/io/vertx/core/impl/BlockedThreadChecker.java

```diff
@@ -45,10 +45,10 @@ public void run() {
           long now = System.nanoTime();
           for (VertxThread thread : threads.keySet()) {
             long execStart = thread.startTime();
-            long dur = now - execStart;
+            long dur = (now - execStart) / 1000000;
```

Comment:
Ah, so I changed ms -> ns because that's the value we use and that is used already in `VertxOptions#maxEventLoopExecuteTime` that is in nanos.